### PR TITLE
Lemon tries his best

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

[Implements kryson's apc sprites, finally, after 1.5 years](https://github.com/tralezab/tgstation/commit/d5177c3daf69296c95752931a91fe4b7374d3919)

There's a few behavior changes in here, something to do with when the
panel needs to be open/closed and cells, something about blocking wires? 
I genuinely don't remember, check before you pr to master, though I doubt anyone will notice.

APC code has broken me

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Not everything works, gimme a sec

**Comment by LemonInTheDark** [comment]
Done

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -366,7 +371,12 @@
 
 	. = ..()
 	// And now, separately for cleanness, the lighting changing
-	if(!update_state)
+	if(update_overlay & UPSTATE_BLUESCREEN)
+		set_light_color(LIGHT_COLOR_BLUE)
+		set_light(lon_range)
```
Comment (dragomagol) [inline code/modules/power/apc.dm:376]: Is this supposed to be `light_on_range`?

**Comment by LemonInTheDark** [comment]
Moved to the org

**Pull request closed without merging**